spec/bump/cli_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "spec_helper"
+
+describe Bump::CLI do
+  around(:each) do |example|
+    modified_path = File.expand_path("../../..", __FILE__) + "/exe:" + ENV["PATH"]
+    ClimateControl.modify PATH: modified_path do
+      example.run
+    end
+  end
+end
```

Comment:
Specs disappeared? 👻

Reply:
Oops. Right. I could fully remove the file as the examples will be removed in the next PR, but let's be clean here: I re-add them now.